internal/provider/codeberg/codeberg.go

```
@@ -0,0 +1,664 @@
+	if err := json.NewDecoder(resp.Body).Decode(&result); err != nil {
+		return "", err
+	}
+	if result.HTMLURL != "" {
+		return result.HTMLURL, nil
+	}
+	return fmt.Sprintf("https://%s/%s/%s/issues/%d#issuecomment-%d", host, owner, repo, number, result.ID), nil
+}
+
+// CreateAnonymousDiscussionComment is not supported on Codeberg.
+func (p *CodebergProvider) CreateAnonymousDiscussionComment(owner, repo string, number int, body string) (string, error) {
+	return "", fmt.Errorf("Codeberg does not support GitHub-style Discussions")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)